developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -8,8 +8,8 @@
 Foreman will run with the following requirements (aside from rubygem dependencies):
 
 * Ruby 2.7.x
-* NodeJS 14
-* NPM 6.x
+* NodeJS 18.x
+* NPM 6.x or NPM 8.x with --legacy-peer-deps flag
 * PostgreSQL 13
```

Comment:
Just a general thought: how about we direct to [matrix.json](https://github.com/theforeman/foreman/blob/develop/.github/matrix.json) and possibly `package.json` for these versions? Calling out NPM with legacy peer deps is still a good thing because it's special, but I'd prefer to avoid duplicating other fields.

Reply:
just by writing: "Please see `.github/matrix.json` for the latest supported versions." ?

Reply:
Updated the text